server/src/main/java/org/elasticsearch/rest/action/cat/RestNodesAction.java

```diff
@@ -67,6 +69,10 @@
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
 public class RestNodesAction extends AbstractCatAction {
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+        LogManager.getLogger(RestNodesAction.class));
+    static final String LOCAL_DEPRECATED_MESSAGE = "Deprecated parameter [local] used, it has no effect the API call locality. " +
```

Comment:
Minor wording change (needs reformatting to width):

```suggestion
    static final String LOCAL_DEPRECATED_MESSAGE = "Deprecated parameter [local] used. This parameter does not cause this API to act locally, and should not be used. It will be unsupported in version 8.0.";
```